reasoning-plan.md

```
@@ -0,0 +1,1039 @@
+  "model": "deepseek-reasoner",
+  "reasoning_effort": "high",
+  "myCustomField": "value",
+  "anotherField": 123,
+  "messages": [...]
+}
+```
+
+### Controlling Headers vs Body
+
+| What you want | How to do it |
+|---------------|--------------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)